src/Prince.php

```diff
@@ -149,6 +149,13 @@ class Prince
      */
```

Comment:
Suggest adding some details to this comment (which I assume is where the [API docs](https://yeslogic.github.io/prince-php-wrapper/current/classes/Prince-Prince.html#method___construct) come from) describing the scenarios in which exceptions will be thrown.

Reply:
Good idea, thanks.